README.md

```diff
@@ -430,6 +433,7 @@ Running `lerna` without arguments will show all commands/options.
 - `lerna`: the current version of Lerna being used.
 - `version`: the current version of the repository.
 - `publishConfig.ignore`: an array of globs that won't be included in `lerna updated/publish`. Use this to prevent publishing a new version unnecessarily for changes, such as fixing a `README.md` typo.
+- `bootstrapConfig.ignore`: an glob that won't be bootstrapped when running the `lerna bootstrap` command.
```

Comment:
I think between (1) functionality consistency and (2) the fact that there is a use-case for "permanent" ignore rules, supporting permanent scope rules is a good idea. Principle of least surprise, and all that.